pangea-sdk/v3/service/audit/api.go

```diff
@@ -963,3 +963,61 @@ type DownloadResult struct {
 func (a *audit) DownloadResults(ctx context.Context, input *DownloadRequest) (*pangea.PangeaResponse[DownloadResult], error) {
 	return request.DoPost(ctx, a.Client, "v1/download_results", input, &DownloadResult{})
 }
+
+// @summary Log streaming endpoint
+//
+// @description This API allows 3rd party vendors (like Auth0) to stream events
+// to this endpoint where the structure of the payload varies across different
+// vendors.
+//
+// @operationId audit_post_v1_log_stream
+//
+// @example
+//
+//	type LogStreamEventData struct {
+//		ClientID     string  `json:"client_id"`
+//		Connection   *string `json:"connection,omitempty"`
+//		ConnectionID *string `json:"connection_id,omitempty"`
+//		Date         string  `json:"date"`
+//		Description  string  `json:"description"`
+//		IP           string  `json:"ip"`
+//		Strategy     *string `json:"strategy,omitempty"`
+//		StrategyType *string `json:"strategy_type,omitempty"`
+//		Type         string  `json:"type"`
+//		UserAgent    string  `json:"user_agent"`
+//		UserID       string  `json:"user_id"`
+//	}
+//
+//	type LogStreamEvent struct {
+//		LogID string             `json:"log_id"`
+//		Data  LogStreamEventData `json:"data"`
+//	}
+//
+//	type LogStreamInput struct {
+//		pangea.BaseRequest
+//
+//		Logs []LogStreamEvent `json:"logs"`
```

Comment:
This is what I think we should have in python too, instead of `data: dict`

Reply:
Wait, this is all commented, do we want this?

Reply:
It's all intentional, this is just an example usage snippet.

Reply:
Hmm, example usage shouldn't be in `examples` folder or in tests?

Reply:
This actually does come from the integration test. Regarding `examples/`, I'm only adding one to the Python SDK because ultimately none of this is _really_ meant to be used via SDK so we're only going to add one real example.